tf-psa-crypto/tests/suites/test_suite_psa_crypto.function

```diff
@@ -1234,6 +1234,34 @@ static void interruptible_signverify_get_minmax_completes(uint32_t max_ops,
         *max_completes = 1;
     }
 }
+
+static uint32_t interruptible_key_agreement_get_min_num_ops(size_t key_bits)
+{
+    /* Those values are taken from documentation of mbedtls_ecp_set_max_ops()
+       in ecp.h. Those values can change at any time. */
+    switch (key_bits) {
+        case 256:
+        {
+            const uint32_t min_values[5] = { 208, 208, 160, 136, 124 }; // P-256
+            return min_values[6 - MBEDTLS_ECP_WINDOW_SIZE];
+            break;
+        }
+        case 384:
+        {
+            const uint32_t min_values[5] = { 682, 416, 320, 272, 248 }; // P-384
+            return min_values[6 - MBEDTLS_ECP_WINDOW_SIZE];
+            break;
+        }
+        case 512:
+        case 521:
+        {
+            const uint32_t min_values[5] = { 1364, 832, 640, 544, 496 }; // P-521
+            return min_values[6 - MBEDTLS_ECP_WINDOW_SIZE];
+            break;
+        }
+    }
+    return 0;
```

Comment:
If `key_bits` is an unexpected value, why is 0 a nice thing to return here?

It could help to have a comment before the function definition that explains the purpose of the function.

Reply:
That's kind of what I'm asking... I'm not sure what the intent of this function is. The way it's used doesn't make the intent clear. In what sense does this function return a minimum?

I expect that the resolution of my comment on GitHub should be a comment in the code that explains what's going on, not a fancier calculation.